<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text-To-Speech</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/Iconspeak.png</normaloff>../icons/Iconspeak.png</iconset>
  </property>
  <widget class="QLabel" name="text_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Text :-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="voice_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Voice:-</string>
   </property>
  </widget>
  <widget class="QLabel" name="speed_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Speed:-</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="slow_spd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="normal_spd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fast_spd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>2.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="speech_btn">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>180</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Speech</string>
   </property>
  </widget>
  <widget class="QPushButton" name="download_btn">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>210</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cncl_btn">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>240</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="male">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="female">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
